facade/logstash.go

```diff
@@ -215,7 +215,7 @@ func getFilters(configs []servicedefinition.LogConfig, filterDefs map[string]str
 		for _, filtName := range config.Filters {
 			//do not write duplicate types, logstash doesn't handle this
 			if !utils.StringInSlice(config.Type, *typeFilter) {
-				filters += fmt.Sprintf("\n  if [type] == \"%s\" {\n    %s\n  }\n",
+				filters += fmt.Sprintf("\n  if \"%s\" in [file] {\n    %s\n  }\n",
```

Comment:
Can you explain how this solves the problem?

Typically, the `file` attribute in logstash and the value of `config.Type` are not the same at all.